doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -189,6 +189,17 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0
```

Comment:
```suggestion
.. rst-class:: v2-9-0-nRF54H20-1 v2-9-0 v2-8-0
```
Overlooked this on the first round.

Reply:
Since "v2-9-0-nRF54H20-1" is not yet released and the release date is not yet known, we could keep this as -rc1, or other option could be that I can add this KI in my PR - https://github.com/nrfconnect/sdk-nrf/pull/19745, where the v2-9-0-nRF54H20-1 tag is added and this PR could be closed.
If this PR is merged now, we will have the v2.9.0-nrf54h20-1 tag in the public doc.

Reply:
Changed it back :)